css/base/_settings.scss

```diff
@@ -0,0 +1,9 @@
+$breakpoints: (
+    mobile: 0,
+    tablet: 800px,
+    desktop: 1200px,
+);
+
+
+
+$color-white: #F5F4D7 !default;
```

Comment:
biały kolor to to nie jest :P, poza tym mogles nadpisac zmienna "$color-white" z pliku base/colors, nie musisz tworzyc nowej zmiennej zeby ja nadpisac

Reply:
To chyba nie do końca rozumiem, na czym polega zmiana w tym zakresie :/